src/SmartComponents/CreatePolicy/CreatePolicy.js

```diff
@@ -16,9 +16,16 @@ export const CreatePolicy = ({
     benchmark, osMajorVersion, complianceThreshold, name, profile, refId, selectedRuleRefIds, systemIds
 }) => {
     const history = useHistory();
+    const location = useLocation();
```

Comment:
I'm not sure I really understand react-router-dom, but `location` is provided by `useHistory`:

```suggestion
    const { push, location } = useHistory();
```

Reply:
Make sense!